upgrades/upgrade-1.6-2.0.rst

```diff
@@ -0,0 +1,291 @@
+    ALTER TABLE se_group_roles CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+    ALTER TABLE se_groups CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+    ALTER TABLE se_permissions CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+    ALTER TABLE se_role_settings CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+    ALTER TABLE se_roles CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+    ALTER TABLE se_security_types CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+    ALTER TABLE se_user_groups CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+    ALTER TABLE se_user_roles CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+    ALTER TABLE se_user_settings CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+    ALTER TABLE se_users CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+    ALTER TABLE ta_tags CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+    ALTER TABLE we_analytics CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+    ALTER TABLE we_analytics_domains CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+    ALTER TABLE we_domains CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+    ALTER TABLE me_format_options CHANGE format_key format_key VARCHAR(191) NOT NULL;
+    ALTER TABLE me_collections CHANGE collection_key collection_key VARCHAR(191) DEFAULT NULL;
+    ALTER TABLE me_collection_types CHANGE collection_type_key collection_type_key VARCHAR(191) DEFAULT NULL;
+    ALTER TABLE me_media_types CHANGE name name VARCHAR(191) NOT NULL;
+    ALTER TABLE me_file_versions CHANGE mimeType mimeType VARCHAR(191) DEFAULT NULL;
+    ALTER TABLE se_users CHANGE email email VARCHAR(191) DEFAULT NULL;
+    ALTER TABLE se_role_settings CHANGE settingKey settingKey VARCHAR(191) NOT NULL;
+    ALTER TABLE se_permissions CHANGE context context VARCHAR(191) NOT NULL;
+    ALTER TABLE se_access_controls CHANGE entityClass entityClass VARCHAR(191) NOT NULL;
+    ALTER TABLE ca_categories CHANGE category_key category_key VARCHAR(191) DEFAULT NULL;
+    ALTER TABLE ca_keywords CHANGE keyword keyword VARCHAR(191) NOT NULL;
+    ALTER TABLE ta_tags CHANGE name name VARCHAR(191) NOT NULL;
+    ALTER TABLE we_domains CHANGE url url VARCHAR(191) NOT NULL;
+    ALTER TABLE we_analytics CHANGE webspace_key webspace_key VARCHAR(191) NOT NULL;
+    ALTER TABLE ro_routes CHANGE path path VARCHAR(191) NOT NULL, CHANGE entity_class entity_class VARCHAR(191) NOT NULL, CHANGE entity_id entity_id VARCHAR(191) NOT NULL;
+    ALTER TABLE me_collection_meta CHANGE title title VARCHAR(191) NOT NULL;
+    ALTER TABLE me_file_version_meta CHANGE title title VARCHAR(191) NOT NULL;
+    ALTER TABLE me_file_versions CHANGE name name VARCHAR(191) NOT NULL;
+
+If you are using jackalope with doctrine-dbal instead of jackrabbit you also have to execute the following queries:
```

Comment:
execute the following statements to update the doctrine-dbal tables

Reply:
Replaced `doctrine-dbal` with `jackalope`